packages/registry/src/ports/PoapRegistry/HttpProvider.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface HttpProvider {
+  request(requestInput: {
+    endpoint: string;
+    method: string;
+    body: any;
+    headers: any;
+  }): Promise<any>;
```

Comment:
```suggestion
  request<R = any>(requestInput: {
    endpoint: string;
    method: string;
    body: any;
    headers: any;
  }): Promise<R>;
```

Reply:
And we know that the results from hasura are not `any` but `{ data: { [index: string]: T } }`.